documentation/src/test/java/org/hibernate/search/documentation/mapper/orm/bridge/dependencies/containers/property/DependenciesContainersPropertyIT.java

```diff
@@ -59,11 +59,11 @@ public void smoke() {
 			SearchSession searchSession = Search.session( entityManager );
 
 			List<Book> result = searchSession.search( Book.class )
-					.where( f -> f.bool()
-							.must( f.match().field( "editionsForSale" ).matching( "paperback" ) )
-							.must( f.match().field( "editionsForSale" ).matching( "kindle" ) )
-					)
-					.fetchHits( 20 );
+					.where( f -> f.and(
+									f.match().field( "editionsForSale" ).matching( "paperback" ),
+									f.match().field( "editionsForSale" ).matching( "kindle" )
+							)
+					).fetchHits( 20 );
```

Comment:
sure, will do. I think that none of the snippets that changed in the docs ware referenced in the guide. 

BTW would your IDE also add these extra indents too? or I'm just not using the same style config 🙈
HSEARCH-4811 would help probably help 😃

Reply:
> BTW would your IDE also add these extra indents too?

I think it would, but only if `.fetchHits` is on the same line as `)`.

I.e. you get extra indents with this:

```
.where( f.and(
...
) ).fetchHits(...)
```

but not with this:


```
.where( f.and(
...
) )
.fetchHits(...)
```